cmd/collector/app/sanitizer/service_name_sanitizer_test.go

```diff
@@ -40,7 +40,7 @@ func (d *fixedMappingCache) Get(key string) string {
 	return k
 }
 
-func (*fixedMappingCache) Put(key string, value string) error {
+func (*fixedMappingCache) Put(string /* key */, string /* value */) error {
```

Comment:
why did you change the order? Seems it would've been better to just comment the name in place, to keep the overall Go syntax of `name type`

Reply:
I thought i would maintain consistency everywhere by using it after the type.

Reply:
@yurishkuro Do you want me to keep the comment before the ones which the formatter doesn't cause an issue

Reply:
no, it's fine. I am not ignoring this PR, it's just too large, I am reviewing it in several tries.

One thing that I found puzzling is why you sometimes use `_`, sometimes no-name, and sometimes replace `_` with no-name? My preference would be to have fewer variations, e.g.
  * always use `_` to indicate unused parameter (I find it more readable this way then just a type)
  * when primitive types are used, add a comment with parameter name

Reply:
> no, it's fine. I am not ignoring this PR, it's just too large, I am reviewing it in several tries.

Sorry, I was just curious if I needed to change something or did something wrong. 

> One thing that I found puzzling is why you sometimes use `_`, sometimes no-name, and sometimes replace `_` with no-name? My preference would be to have fewer variations, e.g.
> 

I use ( _ ) only if some of the parameters not used in the function. If all the parameters are not used I remove all the parameter names instead of using ( _ ).

Reply:
> * always use `_` to indicate unused parameter (I find it more readable this way then just a type)
> * when primitive types are used, add a comment with parameter name

a) This works as well, let me know if I need to make this change
b) I think this is already done

Reply:
> let me know if I need to make this change

at this point I would just leave it as is. I don't want to re-review files I already reviewed.

Reply:
I guess I should have gone with the ( _ ) for everything from the start. My bad.